website/releases/2025-06-11-3.0.0-release.md

```diff
@@ -0,0 +1,63 @@
+---
+title: MLflow 3.0.0
+slug: 3.0.0
+authors: [mlflow-maintainers]
+---
+
+# MLflow 3.0
+
+The open source MLflow community has reached a major milestone. Today, we're releasing MLflow 3.0, which brings production-ready generative AI capabilities to the platform that millions of developers trust for ML operations.
+
+This isn't just another feature update. MLflow 3.0 fundamentally expands what's possible with open source ML tooling, addressing the observability and quality challenges that have made GenAI deployment feel like a leap of faith.
+
+## Major Updates
+
+### 🎯 MLflow `LoggedModel`
+
+MLflow 3.0 introduces a refined architecture with the new `LoggedModel` entity as a first-class citizen, moving beyond the traditional run-centric approach. This enables better organization and comparison of GenAI agents, deep learning checkpoints, and model variants across experiments.
+
+Learn more about [MLflow `LoggedModel`](https://mlflow.org/docs/latest/genai/data-model/#mlflow-loggedmodel-model-management) in the documentation.
+
+### 🔗 Strong Lineage Support
+
+Enhanced model tracking provides comprehensive lineage between models, runs, traces, prompts, and evaluation metrics. The new model-centric design allows you to group traces and metrics from interactive queries and automated evaluation jobs, enabling rich comparisons across model versions.
+
+### ⚡ Prompt Optimization
+
+The MLflow Prompt Registry now includes prompt optimization capabilities, allowing you to automatically improve prompts using evaluation feedback and labeled datasets. This includes versioning, tracking, and systematic prompt engineering workflows.
+
+Learn more about [prompt optimization](https://mlflow.org/docs/latest/genai/prompt-version-mgmt/prompt-registry/optimize-prompts) in the documentation.
+
+### 📚 Revamped Documentation
+
+The MLflow documentation has been completely revamped and is now structured around two primary user journeys: GenAI development and classic machine learning workflows. The new modular organization features dedicated sections for getting started, GenAI-specific capabilities (such as LLMs, prompt engineering, and tracing), and traditional ML features including experiment tracking, model registry, deployment, and evaluation.
```

Comment:
`traditional ML features including experiment tracking, model registry, deployment, and evaluation.` sounds a bit strange. experiment tracking is useful for GenAI too.